packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -12,7 +27,19 @@ test.describe('Call Composite E2E Tests', () => {
       await pages[idx].waitForSelector(dataUiId('call-composite-device-settings'));
       await pages[idx].waitForSelector(dataUiId('call-composite-local-preview'));
       await pages[idx].waitForSelector(`${dataUiId('call-composite-start-call-button')}[data-is-focusable="true"]`);
-      expect(await pages[idx].screenshot()).toMatchSnapshot(`page-${idx}-call-screen.png`);
+      await stubLocalCameraName(pages[idx]);
+      expect(await pages[idx].screenshot()).toMatchSnapshot(`page-${idx}-call-screen.png`, { threshold: 0.5 });
```

Comment:
for threshold, is this something we decide per test or will we apply .5 to all tests? (what does 0.5 really mean?)

Reply:
Ah looks to be a guess and hope value 😅 https://github.com/mapbox/pixelmatch#api
What was venturing beyond the threshold in these tests?

Reply:
Tweaked the code and it doesn't have a need for threshold. (For now)